image/save/distributionpkg/proxy/proxyauth.go

```diff
@@ -82,3 +84,13 @@ func getAuthURLs(remoteURL string) ([]string, error) {
 
 	return authURLs, nil
 }
+
+func ping(manager challenge.Manager, endpoint, versionHeader string) error {
+	resp, err := http.Get(endpoint)
```

Comment:
Here is unsafe

Reply:
you can skip the security action with '// #nosec',
or will it a safe way, reference: https://securego.io/docs/rules/g107.html